app/src/main/java/com/termux/x11/MainActivity.java

```diff
@@ -176,7 +176,23 @@ protected void onCreate(Bundle savedInstanceState) {
         mInputHandler = new TouchInputHandler(this, new RenderStub.NullStub() {
             @Override
             public void swipeDown() {
-                toggleExtraKeys();
+                if (SamsungDexUtils.checkDeXEnabled(getApplicationContext())) {
+                    lorieView.releasePointerCapture();
+                    lorieParent.releasePointerCapture();
```

Comment:
No need to ask for pointer release twice. Pointer capture is activity-wide.

Reply:
I just copied from somewhere else without much thinking about it. Feel free to amend if you think appropriate